jwst/cube_build/src/cube_match_sky_driz.c

```diff
@@ -268,19 +268,6 @@ int match_driz(double *xc, double *yc, double *zc,
 
       // estimate the wavelength overlapping region using max_cdelt3 and max_dwave
       // estimating wavelength range works if we have a linear wavelength
-      if (linear == 1){
```

Comment:
Maybe @jemorrison could clarify why this is there to begin with. Quick blame points to https://github.com/spacetelescope/jwst/pull/6297